zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/processinstance/ProcessInstanceRecord.java

```diff
@@ -169,6 +185,57 @@ public ProcessInstanceRecord setBpmnEventType(final BpmnEventType bpmnEventType)
     return this;
```

Comment:
❌ we should also set the new values in the `wrap()` method

Reply:
Actually we don't want to pass it around, I would leave this out. I can add a comment that this is on purpose.